modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -3,58 +3,74 @@ ifndef::env-site,env-github[]
+
+You can use this workflow for OAS and RAML API specifications. It requires Studio 7.8.x or later, and Mule runtime engine 4.1.4 or later.
 
-You can use this workflow for OAS 2.0 and 3.0 and RAML specifications. It requires Studio 7.8.x or later, and Mule runtime engine 4.1.4 or later.
+From Studio 7.18 and later, you can xref:import-api-specification-exchange.adoc[import an AsyncAPI specification from Exchange] and implement it in your Mule project.
 
 == API Specifications as Exchange Dependencies
 
-You can manage APIs for your Mule application as you manage the modules that compose your application. By leveraging Exchange as your unified repository for all your Mule application dependencies, you can monitor the latest versions of your dependencies. This ensures that your application is up-to-date with the latest API requirements.
+Manage APIs for your Mule application as you manage the modules that compose your application. By leveraging Exchange as your unified repository for all your Mule application dependencies, you can check the latest versions of your dependencies. This ensures that your application is up-to-date with the latest API requirements.
 
 Because the implementation of your API can change with each iteration, you must keep track so that you can update your Mule application to match newer specifications. To help you track these changes, treat your API specifications as dependencies of your project. This way, every time the specification design changes, the API publishes a new API version that you can manage from Studio. You can also update your application to match the new API version.
 
-Each API version that you import from Exchange isn't stored as a local copy on your project’s structure, but as a reference to the API version dependency in Exchange. By referencing the API version in Exchange, Studio can periodically check the status of the version and alert you when a new version is available.
+Each API version that you import from Exchange isn’t stored as a local copy on your project’s structure but as a reference to the API version dependency in Exchange. By referencing the API version in Exchange, Studio can periodically check the status of the version and notify you when a new version is available.
+
+=== Scaffold AsyncAPI, RAML, and OAS API Specifications
+
+Design your API using AsyncAPI, OAS, or RAML specifications in Anypoint Design Center:
+
+* xref:design-center::design-create-publish-api-visual-editor.adoc[]
+* xref:design-center::design-create-publish-api-raml-editor.adoc[]
 
-=== Scaffold RAML and OAS Specifications
+[NOTE]
+You can design AsyncAPI in Design Center with the xref:design-center::design-create-publish-api-raml-editor.adoc[Text Editor]. 
```

Comment:
Missing a word here? "an AsyncAPI API"? "an async API"? "using AsyncAPI"?

Reply:
🤦 I didn't see that! Great catch! Thanks!